### DISCUSSION TITLE
AI少女「霏霏」今天上線服務 桃捷將出現首座無人車站

### DISCUSSION THREAD

**Discussion by xCk27x** [1101-GS3073-C/Python-Community-GS3073-C#500] [General]
[AI少女霏霏](https://udn.com/news/story/7324/6832454)
【螢幕上同時會出現動漫少女「霏霏」和民眾互動，進一步查詢時刻表、票價、路網圖、交通轉乘、周邊景點、天氣資訊、車站資訊、乘車須知、最新消息等資訊】
感覺超有趣，支援超過5種以上的語言，也讓A20興南站成為桃捷首座無人服務車站，之後經過桃園捷運一定要去看看。

**Comment by williamyen0623** [comment]
最近在騎車路上有經過A20
沒想到是這麼酷的車站
不得不說這種以無人自動化技術來深入人類日常生活的科技十分令人興奮

**Comment by brian431** [comment]
雖然說感覺蠻有趣的，不過在人物的設計及建模上還有很大的進步空間。

**Comment by lx3211** [comment]
沒有服務人員的話，會不會有些業務是AI無法處理的呢

**Comment by NcuJerryKao** [comment]
沒人監督 若是突發狀況 該當如何
坦若有的問題 是AI無法發現的 無法解決的 那就糟了

**Comment by BingRong1217** [comment]
節省蠻多人力的

**Comment by Yilin20021016** [comment]
感覺這會令很多人去用這個功能 可以解決很多東西

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
感覺挺有趣的，但也不是每個人都能接受

**Comment by awehao** [comment]
我覺得就像，自動化服務，雖然害事會留下紀錄，但也給了消費者一定程度的隱私定程度的隱私

**Comment by skyling0299** [comment]
讓我想到Amazon的無人商店，透過感應器達成各種目標讓人覺得很有科技感~

**Comment by Xuewen628** [comment]
想到生活中有這種科技逐漸進步的消息我就覺得很有趣 希望在未來能有更多方便的科技來幫助我們取得科技

**Comment by awkward-willy** [comment]
這樣的服務除了方便顧客處理疑難雜症，也能讓使用者擁有與眾不同的體驗，刺激大家使用的興致。在帶有科技感之餘，也兼具實用性以及觀光功能。

**Comment by lukiea** [comment]
紀伯倫曾經說過，信仰是心中的綠洲，思想的駱駝隊是永遠走不到的。這句話令我不禁感慨問題的迫切性。這樣看來，帶著這些問題，我們一起來審視ai 少女。

**Comment by larry40305** [comment]
現在AI應該還沒辦法完全處理所有的問題，應該還是要有真人來輔助
